zipline/data/us_equity_loader.py

```diff
@@ -126,6 +127,12 @@ def _get_adjustments_in_range(self, asset, dts, field):
             The days for which adjustment data is needed.
         field : str
             OHLCV field for which to get the adjustments.
+        is_perspective_after : bool
+            see: `USEquityHistoryLoader.history`
+            If True, the index at which the Mulitply object is registered to
```

Comment:
This is pretty hard to parse, I've read it five times and still am not totally sure what you are saying.  Can you break it up into smaller sentences?

also, typo "Mulitply"